README.md

```diff
@@ -19,7 +19,12 @@ Here is an example of a Grafana dashboard for an ESP project. This dashboard rel
 
 ## Install a Released Version of the Plug-in
 
-Use the provided installation script to install the plug-in:
+An installation script is provided to install the plugin and configure Grafana. The installation script will modify the Grafana deployment by adding an environment variable to enable the plugin
```

Comment:
Are you referring to "GF_INSTALL_PLUGINS"? So we are using an en var that is used by Grafana rather then adding one

Reply:
yes, I should be more explicit about that then. There is potential to overwrite a users existing plugins.